### ISSUE TITLE
Note folder panel size too big when showing note folders as buttons

### ISSUE THREAD

**Issue by davuses** [pbek/QOwnNotes#2749]
Note folder panel is taking up a large space when showing note folders as buttons, the minimum height is too large: 

![image](https://user-images.githubusercontent.com/54793121/225623765-38d460f6-041e-4640-9ba9-046e2a1ea016.png)

It looks like the padding is too large, I have tried to search the style in `style.qss` for this widget but failed to locate it, is there a way to maybe shrink it down?

**Comment by pbek** [comment]
I can't think of an easy way. You would probably give them some object names to reference in a stylesheet in the source code.
They are created here:

https://github.com/pbek/QOwnNotes/blob/796d0295b7e9550c98dbef4fdd4e8042ba400db9/src/mainwindow.cpp#L1868-L1881

**Comment by pbek** [comment]
I guess something else is causing the margin then... (did you already consult https://doc.qt.io/qt-5/stylesheet-reference.html?)

But you should be able to resize the panel anyway...

<img width="245" alt="image" src="https://user-images.githubusercontent.com/1798101/225854388-9df3cfab-3ad8-4a01-9920-f7b45c833a8f.png">

**Comment by davuses** [comment]
> 

I failed to mention that I'm running QOwnNotes on Windows. It could be that the UI layout varied in different platforms, but this is as smallest as I can get the panel to be by reszing it:
![image](https://user-images.githubusercontent.com/54793121/225910676-c450ab9f-e3e2-4ecc-8073-6e4461972351.png)

I could confirm this when I switch to Linux.

**Comment by pbek** [comment]
Can't you move them when you unlock the panels in QOwnNotes?

**Comment by davuses** [comment]
> Can't you move them when you unlock the panels in QOwnNotes?

I can, but no matter where I move the panel it still couldn't get smaller:

![image](https://user-images.githubusercontent.com/54793121/225921272-07baa8e1-681a-4ae5-ab19-2b19faf7e414.png)

**Comment by pbek** [comment]
Under windows everything seems to have another margin/padding. I wonder why that is... maybe some min-height?

**Comment by davuses** [comment]
> Under windows everything seems to have another margin/padding. I wonder why that is... maybe some min-height?

It seems like the extra margin/pddding also appears on Ubuntu 22.04, the desktop environment is Gnome 42.5:
![image](https://user-images.githubusercontent.com/54793121/226163175-c8e0b498-3a72-4d12-916a-d6201d888550.png)

**Comment by pbek** [comment]
What happens if you play with the interface settings?

![image](https://user-images.githubusercontent.com/1798101/226165120-521bacdf-bb62-4ad0-9fef-4aea5785c003.png)

**Comment by davuses** [comment]
> What happens if you play with the interface settings?

I've tried all the style options, but still no luck:

![image](https://user-images.githubusercontent.com/54793121/226165519-85851671-b1fc-45c5-b305-9487367f2456.png)

**Comment by anmsh** [comment]
I put the Note Tags panel at the bottom to get around this:

Single tag:
![image](https://user-images.githubusercontent.com/1550696/226268858-6535f4f1-1df1-4dec-9996-aebaa184a62d.png)

Multiple tags:
![image](https://user-images.githubusercontent.com/1550696/226269027-9aa0ffd4-9cdf-4009-81df-dd6b28c5f659.png)

**Comment by davuses** [comment]
> I put the Note Tags panel at the bottom to get around this:
> 
> Single tag: ![image](https://user-images.githubusercontent.com/1550696/226268858-6535f4f1-1df1-4dec-9996-aebaa184a62d.png)
> 
> Multiple tags: ![image](https://user-images.githubusercontent.com/1550696/226269027-9aa0ffd4-9cdf-4009-81df-dd6b28c5f659.png)

The problem is with Note folder panel, not Tag panel

**Comment by anmsh** [comment]
@davuses oh, misread it. I did not know that Note Folders can be buttons instead of a list. I tried that and there seems to be some margin below the buttons, but not as big as yours. I am on Windows too.

I tried all interface settings too. Everytime I changed something, I needed to drag the panel below Note Folders slightly. That extra padding remained in each case.

![image](https://user-images.githubusercontent.com/1550696/226646065-86942a35-e733-40ed-8aa3-a4caf0730f8d.png)
